actions/status_actions.ts

```diff
@@ -118,14 +117,16 @@ export function loadProfilesMissingStatus(users: UserProfile[]) {
 let intervalId: NodeJS.Timeout;
 
 export function startPeriodicStatusUpdates() {
-    clearInterval(intervalId);
-
-    intervalId = setInterval(
-        () => {
-            store.dispatch(loadStatusesForChannelAndSidebar());
-        },
-        Constants.STATUS_INTERVAL,
-    );
+    return (dispatch: DispatchFunc) => {
```

Comment:
```const posts = postsInChannel.slice(0, (state as GlobalState).views.channel.postVisibility[channelId] || 0);```

this can be a selector ideally :)

Reply:
@nevyangelova  I am a little confused here. Are you suggesting converting this action into a selector?